### ISSUE TITLE
Mapping from EDAM to PSI-MS

### ISSUE THREAD

**Issue by joncison** [edamontology/edamontology#399]
There are some overlaps between EDAM and PSI-MS ... some operations I think, and some high-level data concepts in PSI-MS.

Produce a mapping via equivalence axioms.

This came up in context of the proteomics tools collection publication (which mentions EDAM)

**Comment by joncison** [comment]
@matuskalas check out their coverage of MS file formats, they have nice coverage.

we should at least crosslink to them / define equivalence statements

**Comment by matuskalas** [comment]
Yes sure, looks great, and we should reference them.

Using rdfs:seeAlso like in all other cases. Equivalence is too dangerous (could break some reasoning if inconsistencies beyond our control appear), and not needed in usual applications we have (special applications requiring it can easily add it programmatically whenever they need)

**Comment by rioualen** [comment]
Is this issue still relevant? If so, can you provide more detail?

**Comment by veitveit** [comment]
This would be nice to have. For example mapping the file formats between the two ontologies, as well as some of the data types.

See https://www.ebi.ac.uk/ols4/ontologies/ms

One just would need somebody doing this.

**Comment by hechth** [comment]
@rioualen yeah, this is still relevant. There are many things mapped, though the level of detail of the ms ontology is much higher than EDAM. So there are definitely many terms that can/should be mirrored, but whether the same level of complexity is really needed to capture tool operations is questionable.

**Comment by rioualen** [comment]
Hi, 
Is there a specific list of terms to be added to/mirrored in EDAM, or are you looking for a wider mapping process?
